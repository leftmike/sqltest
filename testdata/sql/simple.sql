--
-- A simple test to start with.
--

DROP TABLE IF EXISTS tbl;

CREATE TABLE tbl (c1 INTEGER, c2 INTEGER, c3 INTEGER);

INSERT INTO tbl VALUES
    (1, 2, 3),
    (7, 8, 9),
    (4, 5, 6);

/*
SELECT d1, d2, d3 FROM table;
*/

SELECT c1, c2, c3 FROM tbl;
