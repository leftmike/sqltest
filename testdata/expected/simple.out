--
-- A simple test to start with.
--
DROP TABLE IF EXISTS tbl
CREATE TABLE tbl (c1 INTEGER, c2 INTEGER, c3 INTEGER)
INSERT INTO tbl VALUES
    (1, 2, 3),
    (7, 8, 9),
    (4, 5, 6)
SELECT * FROM tbl
   c1 c2 c3
   -- -- --
 1  1  2  3
 2  4  5  6
 3  7  8  9
(3 rows)
